docs/content.zh/docs/deployment/advanced/logging.md

```diff
@@ -40,6 +40,21 @@ Flink 中的日志记录是使用 [SLF4J](http://www.slf4j.org/) 日志接口实
 
 <a name="configuring-log4j-2"></a>
 
+### Structured logging
+
+Flink adds the following fields to [MDC](https://www.slf4j.org/api/org/slf4j/MDC.html) of most of the relevant log messages (experimental feature):
+- Job ID
+    - key: `flink-job-id`
```

Comment:
Can we make the key configurable? Users may have different requirements; like wanting to follow ECS when using the json layout where the appropriate key would be `flink.job.id`

Reply:
My reasoning was that users configure the mappings anyways, so making it configurable isn't strictly necessary.
OTH, adding more configuration to Flink makes it more complex; so chose to have it static.
I can add the configuration though if you think it's necessary.

Reply:
With something like this?
```
appender.console.json.eventTemplateAdditionalField[0].type = EventTemplateAdditionalField
appender.console.json.eventTemplateAdditionalField[0].key = flink.job.id
appender.console.json.eventTemplateAdditionalField[0].value = {"$resolver": "mdc", "key": "flink-job-id"}
```

Reply:
That should already be doable - without changes in code, no?